cmd/horcrux/cmd/cosigner.go

```diff
@@ -215,5 +222,7 @@ func StartCosignerCmd() *cobra.Command {
 	}
 
 	cmd.Flags().BoolP("single", "s", false, "set to start horcrux as a single signer")
+	cmd.Flags().StringP("timeout", "t", "1s", "configure cosigner rpc server timeout value, "+
```

Comment:
Ideally we add this to the config file and set a sane default (1500ms) when running `config init`

Reply:
Done! Added this to `cmd/config` instead and took your above recommended changes into account while doing so.
The flag will take in an int  representing the timeout value in milliseconds with a default value of 1500ms